src/transformers/models/segformer/modeling_segformer.py

```diff
@@ -491,7 +491,7 @@ def forward(self, pixel_values, output_attentions=None, output_hidden_states=Non
         >>> import requests
 
         >>> feature_extractor = SegformerFeatureExtractor.from_pretrained("nvidia/segformer-b0-finetuned-ade-512-512")
-        >>> model = SegformerModel("nvidia/segformer-b0-finetuned-ade-512-512")
+        >>> model = SegformerModel.from_pretrained("nvidia/segformer-b0-finetuned-ade-512-512")
```

Comment:
```suggestion
        >>> model = SegformerModel.from_pretrained("nvidia/mit-b0")
```
Can you update the feature extractor also to "nvidia/mit-b0"?

It makes more sense (the pre-trained backbone only).

Reply:
Sure - done.